.changeset/vast-paths-begin.md

```diff
@@ -0,0 +1,17 @@
+---
+"@fluidframework/datastore": minor
+"@fluidframework/test-runtime-utils": minor
+---
+---
+"section": legacy
```

Comment:
```suggestion
"section": legacy
includeInReleaseNotes: false
```

Since these new APIs are only intended for internal use, I suggest noting the changes in the changelogs but not the release notes. If there is more customer impact that I'm not seeing, then I suggest updating the description to focus on how customers should adapt to these changes or what they should do in response.